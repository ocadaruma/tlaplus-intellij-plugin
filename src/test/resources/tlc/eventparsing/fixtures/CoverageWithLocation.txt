TLCEvent.TextEvent(text=/path/to/bin/java -Dfile.encoding=UTF-8 -classpath /path/to/build/idea-sandbox/plugins/tlaplus-intellij-plugin/lib/tla2tools.jar tlc2.TLC -tool -modelcheck -coverage 1 /path/to/CoverageWithLocation.tla)
TLCEvent.TextEvent(text=TLC2 Version 2.18 of Day Month 20?? (rev: 1cec860))
TLCEvent.TLCError(severity=Warning, errors=[TLCEvent.TLCError.SimpleErrorItem(message=Please run the Java VM, which executes TLC with a throughput optimized garbage collector, by passing the "-XX:+UseParallelGC" property.)])
TLCEvent.MC(lines=[Running breadth-first search Model-Checking with fp 66 and seed 5026179315016811554 with 1 worker on 8 cores with 6144MB heap and 64MB offheap memory [pid: 4591] (Mac OS X 12.6 aarch64, Eclipse Adoptium 11.0.15 x86_64, MSBDiskFPSet, DiskStateQueue).])
TLCEvent.TextEvent(text=Starting SANY...)
TLCEvent.SANYStart.INSTANCE
TLCEvent.TextEvent(text=SANY finished.)
TLCEvent.SANYEnd(errors=[])
TLCEvent.TLCStart(startedAt=2022-12-19T21:44:13)
TLCEvent.InitialStatesComputing.INSTANCE
TLCEvent.Progress(timestamp=2022-12-19T21:44:13, diameter=0, total=1, distinct=1, queueSize=1)
TLCEvent.TLCSuccess(fingerprintCollisionProbability=6.5E-19)
TLCEvent.TextEvent(text=The coverage statistics at 2022-12-19 21:44:13)
TLCEvent.CoverageInit(item=TLCEvent.CoverageItem(module=CoverageWithLocation, action=Init, total=1, distinct=1, range=(SourceLocation(line=2, col=0),SourceLocation(line=2, col=4))))
TLCEvent.TextEvent(text=  line 3, col 9 to line 3, col 14 of module CoverageWithLocation: 1)
TLCEvent.CoverageNext(item=TLCEvent.CoverageItem(module=CoverageWithLocation, action=Next, total=6, distinct=2, range=(SourceLocation(line=3, col=0),SourceLocation(line=3, col=4))))
TLCEvent.TextEvent(text=  line 4, col 30 to line 4, col 35 of module CoverageWithLocation: 6)
TLCEvent.TextEvent(text=End of statistics.)
TLCEvent.Progress(timestamp=2022-12-19T21:44:13, diameter=2, total=7, distinct=3, queueSize=0)
TLCEvent.TextEvent(text=7 states generated, 3 distinct states found, 0 states left on queue.)
TLCEvent.TextEvent(text=The depth of the complete state graph search is 2.)
TLCEvent.TextEvent(text=The average outdegree of the complete state graph is 1 (minimum is 0, the maximum 2 and the 95th percentile is 2).)
TLCEvent.TLCFinished(duration=PT0.491S, finishedAt=2022-12-19T21:44:13)
