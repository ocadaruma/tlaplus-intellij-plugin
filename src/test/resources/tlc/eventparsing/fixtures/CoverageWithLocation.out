/path/to/bin/java -Dfile.encoding=UTF-8 -classpath /path/to/build/idea-sandbox/plugins/tlaplus-intellij-plugin/lib/tla2tools.jar tlc2.TLC -tool -modelcheck -coverage 1 /path/to/CoverageWithLocation.tla
@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.18 of Day Month 20?? (rev: 1cec860)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2401:3 @!@!@
Please run the Java VM, which executes TLC with a throughput optimized garbage collector, by passing the "-XX:+UseParallelGC" property.
@!@!@ENDMSG 2401 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 66 and seed 5026179315016811554 with 1 worker on 8 cores with 6144MB heap and 64MB offheap memory [pid: 4591] (Mac OS X 12.6 aarch64, Eclipse Adoptium 11.0.15 x86_64, MSBDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file /path/to/CoverageWithLocation.tla
Semantic processing of module CoverageWithLocation
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2022-12-19 21:44:13)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 1 distinct state generated at 2022-12-19 21:44:13.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2193:0 @!@!@
Model checking completed. No error has been found.
  Estimates of the probability that TLC did not check all reachable states
  because two distinct states had the same fingerprint:
  calculated (optimistic):  val = 6.5E-19
@!@!@ENDMSG 2193 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2022-12-19 21:44:13
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 3, col 1 to line 3, col 4 of module CoverageWithLocation>: 1:1
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 3, col 9 to line 3, col 14 of module CoverageWithLocation: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Next line 4, col 1 to line 4, col 4 of module CoverageWithLocation (4 30 4 35)>: 2:6
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 4, col 30 to line 4, col 35 of module CoverageWithLocation: 6
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(2) at 2022-12-19 21:44:13: 7 states generated (860 s/min), 3 distinct states found (368 ds/min), 0 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
7 states generated, 3 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2194:0 @!@!@
The depth of the complete state graph search is 2.
@!@!@ENDMSG 2194 @!@!@
@!@!@STARTMSG 2268:0 @!@!@
The average outdegree of the complete state graph is 1 (minimum is 0, the maximum 2 and the 95th percentile is 2).
@!@!@ENDMSG 2268 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 491ms at (2022-12-19 21:44:13)
@!@!@ENDMSG 2186 @!@!@
